src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6600,4 +6600,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_FunctionPointerTypesInAttributeNotSupported" xml:space="preserve">
     <value>Using a function pointer type in a 'typeof' in an attribute is not supported.</value>
   </data>
+  <data name="IDS_FeatureImprovedInterpolatedStrings" xml:space="preserve">
+    <value>improved interpolated strings</value>
```

Comment:
>improved [](start = 11, length = 8)

How are we going to call the next improvement to interpolated strings? Consider using more specific feature name.

Reply:
I'll rename to `interpolated string builders`

---
In reply to: [610676632](https://github.com/dotnet/roslyn/pull/52422#discussion_r610676632) [](ancestors = 610676632)